Content.Server/_EE/Silicon/Charge/Systems/SiliconChargeSystem.cs

```diff
@@ -198,4 +206,24 @@ private float SiliconHeatEffects(EntityUid silicon, SiliconComponent siliconComp
 
         return 0;
     }
+
+    // TheDen - IPC Dynamic Power draw
+    private float SiliconMovementEffects(EntityUid silicon, SiliconComponent siliconComp)
+    {
+        // Calculate dynamic power draw.
+        if (!TryComp(silicon, out MovementSpeedModifierComponent? movement) ||
+            !TryComp(silicon, out PhysicsComponent? physics) || !TryComp(silicon, out InputMoverComponent? input))
+            return 0;
+
+        if (input.HeldMoveButtons == 0x0 || _jetpack.IsUserFlying(silicon)) // If nothing is being held or jet packing
```

Comment:
Don't use a magic number to compare against an enum
```suggestion
        if (input.HeldMoveButtons == MoveButtons.None || _jetpack.IsUserFlying(silicon)) // If nothing is being held or jet packing
```